src/features/surveys/components/SubmissionWarningAlert.tsx

```diff
@@ -6,7 +6,7 @@ import ZUIFuture from 'zui/ZUIFuture';
 import { Alert, AlertTitle, Box, Link } from '@mui/material';
 
 type SubmissionWarningAlertProps = {
-  campId: number | 'standalone';
+  campId: number | 'standalone' | 'inherited';
```

Comment:
I wonder if "shared" is better than "inherited". What do you think @ziggabyte?

This is to indicate that a parent organization owns this survey, and has shared it with all of it's sub-orgs (including your current organization). It's just visible in the URL, so it's not critical, but we might as well pick whatever we're most comfortable with right now and not have to feel like we should change it later. :blush¨

Reply:
I think "shared" sounds better!